config/proto_conversions_test.go

```diff
@@ -145,6 +145,19 @@ var testNetworkConfig = NetworkConfig{
 		Sessions: SessionsConfig{
 			HeartbeatWindow: 5 * time.Second,
 		},
+		TrafficTunnelEndpoints: []TrafficTunnelEndpoint{
+			{
+				Port:              9090,
+				ConnectionTimeout: 20 * time.Second,
+			},
+			{
+				Port:              27017,
+				ConnectionTimeout: 40 * time.Millisecond,
```

Comment:
same as above

Reply:
Does the `TrafficTunnelEndpoint` right below this one capture what you're looking for? Just a specified port so we use a zero-valued `time.Duration` (`0`) for `ConnectionTimeout`?